.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore
+test.md
```

Comment:
- [ ] **Requirement** There is a .gitignore file listing `test.md` file and `node_modules` directory (as it can be added by accident while setting up linters). Kindly add `node_modules` here.